targets/psa_switch/psa_switch.cpp

```diff
@@ -333,6 +344,16 @@ PsaSwitch::ingress_thread() {
     Deparser *deparser = this->get_deparser("ingress_deparser");
     deparser->deparse(packet.get());
 
+    // resubmit - do not move below multicast
+    Field &f_resubmit = phv->get_field("psa_ingress_output_metadata.resubmit");
+    if (f_resubmit.get_int()) {
+      // since we resubmit the copy & toss original, we update the copy's metadata
+      phv_copy->get_field("psa_ingress_parser_input_metadata.packet_path").set(5);
```

Comment:
I would recommend defining something like `static constexpr int PACKET_PATH_RESUBMIT = 5;` and using the name, in case the number ends up being used more than once it will be easier to find them all later and update them to the new enum encodings

Reply:
It would also be nice to assign the resubmitted packet's "ingress_timestamp" field the value of the current time here, too.  PSA specifies that it should be updated, not merely copied from the original packet that caused the resubmit.

Reply:
We addressed both of these in a later commit